src/test/java/peoplesoft/model/money/PaymentTest.java

```diff
@@ -0,0 +1,220 @@
+    }
+
+    @Test
+    public void removePendingPayments_multiplePerson() {
+        Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        Employment employment = new Employment();
+
+        Person newAlice = new PersonBuilder(ALICE).build();
+        Person newBenson = new PersonBuilder(BENSON).build();
+        Person newCarl = new PersonBuilder(CARL).build();
+
+        employment.associate(EATING, newAlice);
+        employment.associate(RUNNING, newAlice);
+        employment.associate(EATING, newBenson);
+        employment.associate(RUNNING, newCarl);
+
+        Map<ID, Payment> alicePayments = new HashMap<>(newAlice.getPayments());
+        Map<ID, Payment> bensonPayments = new HashMap<>(newBenson.getPayments());
+        Map<ID, Payment> carlPayments = new HashMap<>(newCarl.getPayments());
+
+        PaymentHandler.createPendingPayments(EATING, model, employment);
+        PaymentHandler.createPendingPayments(RUNNING, model, employment);
+
+        PaymentHandler.removePendingPayments(EATING, model, employment);
+
+        Payment newAlicePayment = Payment.createPayment(newAlice, RUNNING, RUNNING.calculatePay(newAlice.getRate()));
+        Payment newCarlPayment = Payment.createPayment(newCarl, RUNNING, RUNNING.calculatePay(newCarl.getRate()));
+
+        alicePayments.put(RUNNING.getJobId(), newAlicePayment);
+        carlPayments.put(RUNNING.getJobId(), newCarlPayment);
+
+        assertEquals(alicePayments, model.getPerson(newAlice.getPersonId()).getPayments());
+        assertEquals(bensonPayments, model.getPerson(newBenson.getPersonId()).getPayments());
+        assertEquals(carlPayments, model.getPerson(newCarl.getPersonId()).getPayments());
+    }
+
+    @Test
+    public void finalizePayments_singlePerson() {
+        Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        Employment employment = new Employment();
+        Person newAlice = new PersonBuilder(ALICE).build();
+
+        employment.associate(EATING, newAlice);
+        employment.associate(RUNNING, newAlice);
+
+        Map<ID, Payment> alicePayments = new HashMap<>(newAlice.getPayments());
+
+        PaymentHandler.createPendingPayments(EATING, model, employment);
+        PaymentHandler.createPendingPayments(RUNNING, model, employment);
+
+        Payment newAliceRunningPayment = Payment.createPayment(
+                newAlice, RUNNING, RUNNING.calculatePay(newAlice.getRate()));
+        Payment newAliceEatingPayment = Payment.createPayment(
+                newAlice, EATING, EATING.calculatePay(newAlice.getRate()));
+        newAliceRunningPayment.pay();
+
+        alicePayments.put(EATING.getJobId(), newAliceEatingPayment);
+        alicePayments.put(RUNNING.getJobId(), newAliceRunningPayment);
+
+        assertEquals(alicePayments, model.getPerson(newAlice.getPersonId()).getPayments());
+    }
+
+    @Test
+    public void finalizePayments_multiplePerson() {
```

Comment:
this test seems like it does the `singlePerson` test twice (as opposed to the other `singlePerson`/`multiplePerson` tests), which might be a bit redundant, but nbd